ddtrace/llmobs/_integrations/google_utils.py

```diff
@@ -11,9 +12,9 @@
 from ddtrace.llmobs._utils import _get_attr
 
 
-# google genai has roles "model" and "user", but in order to stay consistent with other integrations,
-# we use "assistant" as the default role for model messages
-DEFAULT_MODEL_ROLE = "assistant"
+# ---------------------------------------------------------------------------------------------------
+# Below are util functions shared by all google integrations
+# ---------------------------------------------------------------------------------------------------
```

Comment:
```suggestion
```
Your code should be self-explanatory, if you'd really like to enforce structure to the utils file then maybe add a single-line comment like `# utility methods should be suffixed by which integration it supports.`